niworkflows/interfaces/images.py

```diff
@@ -42,6 +42,13 @@ class _RegridToZoomsInputSpec(BaseInterfaceInputSpec):
         usedefault=True,
         desc="clip the data array within the original image's range",
     )
+    smooth = traits.Either(
+        False,
+        traits.Bool(),
+        traits.Float(),
```

Comment:
shouldn't the traits go first here?
```suggestion
        traits.Bool(),
        traits.Float(),
        False,
```

Reply:
I think either takes the first argument for default. To be sure:

```suggestion
        traits.Bool(),
        traits.Float(),
        default=False,
```

(please note that `Either` is not regular and doesn't take `default_value`)